deps.edn

```diff
@@ -13,9 +13,11 @@
                       org.clojure/data.json]}
         org.clojure/tools.logging {:mvn/version "1.1.0"}
         io.github.yetanalytics/project-persephone
-        {:git/sha "3a97133"
-         :git/tag "v0.7.3"
+        {:git/sha "387a11d44eec63b0ee7b2e25f6a2c002eed0fe21"
+         #_"3a97133"
+         #_#_:git/tag "v0.7.3"
```

Comment:
Persephone v0.8.0 is out now, just so you know.

Reply:
Oh, I checked the list and didn't see it

Reply:
I pushed the tag like 5 seconds ago